src/main/webapp/js/userMapData.json

```diff
@@ -1,12 +1,9 @@
 {
-  "lastUpdated": "15 March 2017",
+  "lastUpdated": "28 February 2017",
   "institutes": {
-  	 "Abu Dhabi": [ 	
-  	   "Khalifa University"
```

Comment:
@whipermr5 it was already included in the "UAE" (United Arab Emirates) and we deemed it is better to only remain there since the "Abu Dabi" is part of them. Do you agree with that?